### ISSUE TITLE
Training on my own data, ap = 0

### ISSUE THREAD

**Issue by weisq2691** [endernewton/tf-faster-rcnn#315]
My training set has only one class of targets. In tensorboard, there is no problem in training images, but loss doesn't converge. Finally, ap=0. 
Did anyone meet the same problem?

**Comment by endernewton** [comment]
maybe you need to adjust learning rate.
